app/controllers/declaration/RepresentativeDetailsController.scala

```diff
@@ -64,8 +64,10 @@ class RepresentativeDetailsController @Inject()(
     declarationType match {
       case DeclarationType.SUPPLEMENTARY =>
         controllers.declaration.routes.DeclarationAdditionalActorsController.displayPage
-      case DeclarationType.STANDARD | DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL | DeclarationType.CLEARANCE =>
+      case DeclarationType.STANDARD | DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL =>
         controllers.declaration.routes.CarrierDetailsController.displayPage
+      case DeclarationType.CLEARANCE =>
+        controllers.declaration.routes.DeclarationHolderController.displayPage
```

Comment:
Representative details should lead to carrier details now on clearance (this was part of my ticket 2245)